projects/common/zcu102/zcu102_system_bd.tcl

```diff
@@ -49,12 +49,12 @@ set_property -dict [list \
   CONFIG.PSU__SPI0__PERIPHERAL__IO {EMIO} \
   CONFIG.PSU__SPI0__GRP_SS1__ENABLE 1 \
   CONFIG.PSU__SPI0__GRP_SS2__ENABLE 1 \
-  CONFIG.PSU__CRL_APB__SPI0_REF_CTRL__FREQMHZ 50 \
+  CONFIG.PSU__CRL_APB__SPI0_REF_CTRL__FREQMHZ 100 \
```

Comment:
Can you change this back to the original freq.
If you believe the changes are required, you can add an overwriting setting in
projects/adi_vna_8ch/zcu102/system_bd.tcl
To my knowledge Linux will configure/overwrite this configuration as it is described in devicetree.
But for no-OS this change might break the functionality of some designs.

Reply:
Those frequencies have their original value (100MHz). I had changed them to 50Mhz in one commit, but then changed back to 100MHz again.